components/ui/Footer.js

```diff
@@ -15,41 +54,34 @@ export const Footer = () => {
         </p>
         <p>&copy; 2024 Galactica. All rights reserved.</p>
       </div>
-      {/* TASK - React 1 week 2 */}
-      {/* Create a new List for the Pages */}
-      {/* We need to use the <Link /> component here */}
-      {/* <div className={styles.pages}>
+      <div className={styles.pages}>
         <h3>Pages</h3>
         <ul>
-          <li> <Link/> </li>
-          ...
-        </ul>
-      </div> */}
-      {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
-
-      <div className={styles.footerLinks}>
-        <h3>Follow us</h3>
-        <ul className={styles.footerList}>
-          <li>
-            <a href='https://facebook.com'>Facebook</a>
-          </li>
           <li>
-            <a href='https://instagram.com'>Instagram</a>
+            <Link href='/'>Home</Link>
```

Comment:
Since I can't leave any other comments.
This could be converted to a `.map()` to render it, since you are repeating yourself here 😄

Reply:
Done!